app/assets/stylesheets/components/_body_layout.sass

```diff
@@ -0,0 +1,14 @@
+$BodyLayout-body-spacing: $Theme-spacing-base
+$BodyLayout-color-title: Theme-color(text, title-page)
+$BodyLayout-fontSize-title: Theme-fontSize(title, large, font-size)
+$BodyLayout-fontWeight-title: Theme-fontWeight(bold)
+$BodyLayout-spacing-title: 4 * $BodyLayout-body-spacing
+
+.BodyLayout-header
+  color: $BodyLayout-color-title
+  font-size: $BodyLayout-fontSize-title
+  font-weight: $BodyLayout-fontWeight-title
+  padding: $BodyLayout-spacing-title 0
```

Comment:
I'm not sure how the design is, but I'd say the reason for having so big a spacing on both sides of this "header" isn't really the best. We'll discuss this personally